closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -474,6 +556,235 @@ JNIEXPORT jlong JNICALL Java_jdk_crypto_jniprovider_NativeCrypto_loadCrypto
         }
     }
 
+    if (traceEnabled) {
+        fprintf(stdout, "%s: %s\n", message, openssl_version);
+        fflush(stdout);
+    }
+
+    return ossl_ver;
+}
+
+
+void * load_crypto_library_libname(jboolean traceEnabled, const char *libNameProp) {
```

Comment:
This function can be `static`; suggest `libName` is a better name for the second parameter:
```c
static void *
load_crypto_library_libname(jboolean traceEnabled, const char *libName)
{
```

Reply:
Changed.